services/storehost/extMgr.go

```diff
@@ -105,6 +134,9 @@ type (
 		//    ref is 0, the 'extentLock' needs to be acquired and the ref re-checked
 		//    before initiating clean-up/tear-down
 
+		// count of close/cleanup callbacks that have reference on the structure
+		cleanupRef uint32
```

Comment:
this is a mechanism used by the callback-handlers, if they do not want the extent-context to be torn down. this is not used by the 'extStats' reporter -- which is the only component currently using this; but i plan to add others pretty soon.